docs/resources/mws_ncc_binding.md

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Deployment"
+---
+# databricks_mws_ncc_binding Resource
+
+-> **Note** Initialize provider with `alias = "account"`, `host = "https://accounts.azuredatabricks.net"` and use `provider = databricks.account` for all `databricks_mws_*` resources.
+
+-> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html) in Azure & AWS.
```

Comment:
NCC binding has GA. NCC PL is in public preview (GA in early April).

Reply:
done